lighthouse-core/formatters/partials/table.css

```diff
@@ -43,11 +38,23 @@
 .table_list em + code, .table_list em + pre {
   margin-top: 10px;
 }
-.table_list td img {
+.table_list .table-column {
+  text-align: right;
+}
+.table_list .table-column.table-column-url {
+  text-align: left;
+  width: 250px;
```

Comment:
could add more here, but on smaller screen sizes the optimized images table starts getting pretty squished, min-width doesn't seem to have an impact on fixed table width